pkg_resources/extern/__init__.py

```diff
@@ -78,3 +79,10 @@ def install(self):
     'more_itertools',
 )
 VendorImporter(__name__, names).install()
+
+if TYPE_CHECKING:
+    from . import packaging as packaging
```

Comment:
I only type-stubs the re-imports for `packaging` because that's the only one that was needed for internal use.